### DISCUSSION TITLE
How do I set up Insert mode to basicity be normal VSCode (Vim disabled)?

### DISCUSSION THREAD

**Discussion by MichaelLesirge** [VSCodeVim/Vim#8171] [Q&A]
EDIT: I would close this discussion if I could, I have gotten better at it now.

Hi everyone,

I love the keyboard shortcuts offered in Vim Normal mode and find them very helpful but I also like the keyboard shortcuts in normal VSCode. For example all the multi-cursor stuff. like ctrl+d (highlight next occurrence) or ctrl+f2: (highlight all occurrences).  For some reason whenever I use ctrl+d or ctrl+f2 it appears to work normally but puts me in visual mode even though I have turned off mouse selection goes into Visual mode.

#### User/settings.json:
    "vim.useCtrlKeys": false, //appears to not work for me, I just removed bindings for some stuff with ctrl keys
    "vim.startInInsertMode": true,
    "vim.mouseSelectionGoesIntoVisualMode": false,
    "vim.smartRelativeLine": true,
    "vim.useSystemClipboard": true,

#### User/keybindings.json:
(minus sign in front of command means it is disabled): 
```
    {
        "key": "shift+alt+down",
        "command": "-extension.vim_cmd+alt+down",
        "when": "editorTextFocus && vim.active && !inDebugRepl"
    },
    {
        "key": "shift+alt+up",
        "command": "-extension.vim_cmd+alt+up",
        "when": "editorTextFocus && vim.active && !inDebugRepl"
    },
    {
        "key": "ctrl+d",
        "command": "-extension.vim_ctrl+d",
        "when": "editorTextFocus && vim.active && vim.use<C-d> && !inDebugRepl"
    }
```

**Comment by anthonykaufman** [comment]
You said you've gotten better at this but did you ever find a solution to the ctrl+d performing an Add Next Occurrence when in insert mode?
